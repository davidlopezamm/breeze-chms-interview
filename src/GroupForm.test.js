import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import { MemoryRouter } from 'react-router';
import Groups from './pages/group/edit';
import { Route } from 'react-router-dom';

configure({ adapter: new Adapter() });
let wrapper, data;

describe('<ResultsList /> Group', () => {

    beforeAll(() => {
        wrapper = shallow(<Groups
        match={{params: {id: 1}, isExact: true, path: "", url: ""}} />)
        data = [{
            "id": 132,
            "group_name": "Elders",
            "updated_at": "2019-07-20 22:05:47",
            "created_at": "2019-07-20 22:05:47"
        }]

        wrapper.setState({ 'data' : data });
    });

    test('should match the snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
})
