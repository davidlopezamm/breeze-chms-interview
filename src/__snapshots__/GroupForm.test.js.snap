// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultsList /> Group should match the snapshot 1`] = `
<div>
  <h1>
    Group
  </h1>
  <form
    onSubmit={[Function]}
  >
    <Form
      as="form"
    >
      <FormGroup>
        <FormInput
          as={[Function]}
          control={[Function]}
          label="Group name"
          name="group_name"
          onChange={[Function]}
          placeholder="Name"
          width={5}
        />
      </FormGroup>
    </Form>
    <Button
      as="button"
      type="submit"
    >
      Submit
    </Button>
  </form>
</div>
`;
